presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -341,11 +361,11 @@ void PeriodicTaskManager::updateCacheStats() {
         kCounterSsdCacheCumulativeWrittenBytes,
         memoryCacheStats.ssdStats->bytesWritten);
     RECORD_METRIC_VALUE(
-        kCounterSsdCacheCumulativeCachedEntries,
-        memoryCacheStats.ssdStats->entriesCached);
+        kCounterSsdCacheCumulativeAgedOutEntries,
+        memoryCacheStats.ssdStats->entriesAgedOut)
     RECORD_METRIC_VALUE(
-        kCounterSsdCacheCumulativeCachedBytes,
-        memoryCacheStats.ssdStats->bytesCached);
+        kCounterSsdCacheCumulativeAgedOutRegions,
+        memoryCacheStats.ssdStats->regionsAgedOut);
```

Comment:
IF this about aged out entries - do we only care about them when TTL is on or we always care about them?

Reply:
We only care about them when TTL is on